lib/components/Modal/Modal.css

```diff
@@ -30,45 +28,82 @@
 }
 
 .header {
+  border-radius: 3px 3px 0 0;
+  padding: 23px 24px 4px 24px;
```

Comment:
```suggestion
  padding: 23px var(--vds-spacer-md) 4px var(--vds-spacer-md);
```

Reply:
@avkvirtru thoughts on this?

Reply:
always hooray for [design tokens](https://avkvirtru.github.io/virtuoso-design-system/?path=/docs/basics-design-tokens--page) :)

I would also consider changing 
```css
border-radius: 3px 3px 0 0;
``` 
to
```css
border-radius: 0;
border-top-left-radius: var(--vds-border-radius-xs);
border-top-right-radius: var(--vds-border-radius-xs);
```